torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -1156,6 +1182,8 @@ void scheduleNormalization(
   const auto& in_tv = ir_utils::filterByType<TensorView>(fusion->inputs());
   const auto& out_tv = ir_utils::filterByType<TensorView>(fusion->outputs());
 
+  cacheInputs(fusion, rparams, reduction_tv, other_tv);
```

Comment:
We want the cache afters only in below branch:

```cpp
if (rparams.fastest_dim) {
    const bool kHasOuterAxis = reduction_tv.front()->nDims() > 1;
    if (rparams.persistent_kernel) {
```

Reply:
Yes, that condition is checked inside `cacheInputs`.